RDK-54894,RDK-54893: RT Thread Priority Updates

ddevad opened the pull request:
Reason for change:1) AV metrics header update for
4.9 & 5.4 kernel
2) change drm_mode_rmfb_work_fn workQ to high priority workQ thread that is unbounded
3) Set DRM vblank timeout to 1.5x frame period for 4.9 kernle

Test Procedure: Refer Ticket

Risks: Medium
Priority:P1

Change-Id: I7d74698f73f9cc27ca9385fd88cc9eedebeec1b2
Signed-off-by: Deekshith Devadas <deekshit.devadasy@sky.uk>
(cherry picked from commit 08b0b8158c910fb53b9f2ab0bea18693e427a591)

ddevad committed:
RDK-54894,RDK-54893: RT Thread Priority Updates

Reason for change:1) AV metrics header update for
4.9 & 5.4 kernel
2) change drm_mode_rmfb_work_fn workQ to high priority
workQ thread that is unbounded
3) Set DRM vblank timeout to 1.5x frame period for 4.9 kernle

Test Procedure: Refer Ticket

Risks: Medium
Priority:P1

Change-Id: I7d74698f73f9cc27ca9385fd88cc9eedebeec1b2
Signed-off-by: Deekshith Devadas <deekshit.devadasy@sky.uk>
(cherry picked from commit 08b0b8158c910fb53b9f2ab0bea18693e427a591)

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 1 **violation**, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/meta-rdk-oss-reference/70/rdkcentral/meta-rdk-oss-reference

* Commit: fde1f3d2e086f52c02884b073fab14400088446a

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/75ea316e2a7946bc23c6ec28637dc50e)'

rdkcmf-jenkins commented:
b'## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: fde1f3d2e086f52c02884b073fab14400088446a
'

ddevad committed:
RDKTV-36325: Add missing change from below gerrit commit

RDKTV-29560: Apache-4k Collectd service Failed to load

Reason for change: Change to fix the crash on 4k platforms while loading av metric plugin
Test Procedure: Run collectd to load AV metrics
Risks: Low

Change-Id: I23ef03ab90304b653a18847ec33054473267c07b
Signed-off-by: shashank.kumar@sky.uk <shashank.kumar@sky.uk>
(cherry picked from commit fead9a0873f812df8b3b8297d92d915dc95400d6)

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 1 **violation**, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/meta-rdk-oss-reference/70/rdkcentral/meta-rdk-oss-reference

* Commit: 961f6f4e19ed49afd6341f60099382620e74c70c

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/4dd7805befac872812c23113aaef3236)'

mhughesacn commented:
Hi Deekshith: Please add source info to the new patches.  By this, I mean the source of any new or changed code.  Please use a field "Source: " and Sky/Comcast or an external URL etc.

ddevad committed:
RDKTV-36325: Update Source info in patches

ddevad commented:
> Hi Deekshith: Please add source info to the new patches. By this, I mean the source of any new or changed code. Please use a field "Source: " and Sky/Comcast or an external URL etc.

Done

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 1 **violation**, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/meta-rdk-oss-reference/70/rdkcentral/meta-rdk-oss-reference

* Commit: af613e4dc41afc6598f7c41fd4dc84e8c854f64b

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/6936020169383c4f07f4d24f81335532)'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 1 **violation**, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/meta-rdk-oss-reference/70/rdkcentral/meta-rdk-oss-reference

* Commit: af613e4dc41afc6598f7c41fd4dc84e8c854f64b

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/baad85648be0697009fe62bbe409350c)'

maniselva006c reviewed (commented):
Are these changes specific to a particular SoC? If they are SoC-specific, would these modifications impact streaming devices that utilize different SoCs, such as Realtek and Broadcom?

Considering that OSS releases are platform-independent and architecture-specific, and the source recipes are open-sourced, would these adjustments affect community platforms which might use different soc?

Do these modifications impact the ABI? If so, we should consider incrementing the major version across all stack layers.

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 1 **violation**, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/meta-rdk-oss-reference/70/rdkcentral/meta-rdk-oss-reference

* Commit: 82e8585a786a9b59a91d4faee1685e61ab3b33a0

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/202938b5f64fc561b2d1090dc92d5ea1)'

maniselva006c commented:
Please inclement the linux-libc-header revision from R1 to R2 at https://github.com/rdkcentral/meta-rdk-oss-reference/blob/develop/conf/include/package_revisions_oss.inc#L587

rdkcmf-jenkins commented:
b'## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: 82e8585a786a9b59a91d4faee1685e61ab3b33a0
'

ddevad committed:
RDKTV-36325: Update linux-libc-headers package revision

ddevad commented:
> Please increment the linux-libc-header revision from R1 to R2 at https://github.com/rdkcentral/meta-rdk-oss-reference/blob/develop/conf/include/package_revisions_oss.inc#L587

Done

rdkcmf-jenkins commented:
b'## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: 209e0b3f95648f05db1984a467a4c37cba59551a
'
